xml/depl_maintenance.xml

```diff
@@ -312,6 +312,80 @@
   </orderedlist>
  </sect1>
 
+ <sect1 xml:id="sec.depl.maintenance.recover.compute.node.failure">
+  <title>Recovering from &compnode; Failures</title>
+  <para>
+   The following procedure assumes that there is at least one other &compnode;
+   already running. If this is not the case, refer to <xref
+   linkend="sec.depl.maintenance.bootstrap.compute.plane" />.
+  </para>
+  <procedure>
+   <step>
+    <para>
+     If the &compnode; failed, it should have been fenced.  Verify that this is
+     the case. Otherwise, check <filename>/var/log/pacemaker.log</filename> on
+     the Designated Co-ordinator to determine why the &compnode; was not fenced. The
+     most likely reason is a problem with STONITH devices.
+    </para>
+   </step>
+   <step>
+    <para>
+     Determine the cause of the &compnode;'s failure.
+    </para>
+   </step>
+   <step>
+    <para>
+     Boot the &compnode; again.
+    </para>
+   </step>
+   <step>
+    <para>
+     Check whether the <systemitem>crowbar_join</systemitem> script ran
+     successfully on the &compnode;. If this is not the case,, check the log
+     files to find out the reason.
+    </para>
+   </step>
+   <step>
+    <para>
+     If the <systemitem>chef-client</systemitem> agent triggered by
+     <systemitem>crowbar_join</systemitem> succeeded, the
+     <systemitem>pacemaker_remote</systemitem> service should be up and
+     running. Verify that this is indeed the case.
+    </para>
+   </step>
+   <step>
+    <para>
+     Check whether the remote node is registered and seen as healthy by the
+     core cluster. If this is not the case check
+     <filename>/var/log/pacemaker.log</filename> on the Designated Co-ordinator
+     to determine the cause. There should be a remote primitive running on the
+     core cluster (active/passive). This primitive is responsible for
+     establishing a TCP connection to the
+     <systemitem>pacemaker_remote</systemitem> service on the &compnode;. Use
+     standard network debugging techniques to make sure nothing is preventing
+     this TCP connection from being established (for example, problems with
+     NICs, switches, firewalls etc.).
+    </para>
+   </step>
+   <step>
+    <para>
+     If Pacemaker can communicate with the remote node, it should try to start
+     the <systemitem>nova-compute</systemitem> service on it as part of the
+     cloned group <literal>cl-g-nova-compute</literal>, using the NovaCompute
+     OCF resource agent. (This cloned group will block startup of
+     <systemitem>nova-evacuate</systemitem> until at least one clone is
+     started. This why the next section is necessary as a related but different
+     procedure. See also the next step.
```

Comment:
"See next step"/"See next section" is weird -- please use xrefs if these are useful potentially references (the section reference seems legitimate, but the one to next step seems less legitimate).